src/modules/commander/Commander.cpp

```diff
@@ -765,6 +765,21 @@ Commander::handle_command(vehicle_status_s *status_local, const vehicle_command_
 
 				bool cmd_arms = (static_cast<int>(cmd.param1 + 0.5f) == 1);
 
+				// Arm/disarm is enforced only when param2 is set to a magic number.
+				const bool enforce_in_air = (static_cast<int>(std::round(cmd.param2)) == 21196);
```

Comment:
I'd keep it consistent:
```suggestion
				const bool enforce_in_air = (static_cast<int>(cmd.param2 + 0.5f) == 21196);
```

Reply:
No, I think we should one by one get rid of these strange `0.5f`s. And by introducing it others will see it and adopt it.